ANFOG/RT/move_to_incoming.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# script to loop through realtime subdirectories to find new netcdf files
+# then zip related files together
+INCOMING_RT=$INCOMING_DIR/ANFOG/realtime
+
+for file in `find $INCOMING_RT/. -iname "*FV0*.nc"`; do
+
+     fileNc=$file
+     zipName=`dirname $file`/"`basename $file .nc`.zip"
```

Comment:
I'm not sure what this is trying to do? is `zipName` supposed to be the file name with ".zip" appended?